packages/react/src/components/Slider/Slider.test.tsx

```diff
@@ -34,95 +25,94 @@ describe('Slider', () => {
     expect(slider.current?.value).toEqual(12);
   });
 
   it('can provide the current range', () => {
     const slider = React.createRef<ISlider>();
 
-    wrapper = mount(<Slider defaultValue={12} min={0} max={100} componentRef={slider} ranged />);
+    render(<Slider defaultValue={12} min={0} max={100} componentRef={slider} ranged />);
     expect(slider.current?.range).toEqual([0, 12]);
   });
 
   it('can set id', () => {
-    wrapper = mount(<Slider id="test_id" styles={{ titleLabel: 'test_label' }} />);
+    const { container, getByRole } = render(<Slider id="test_id" styles={{ titleLabel: 'test_label' }} />);
 
-    const sliderSlideBox = wrapper.find('.ms-Slider-slideBox');
-    expect(sliderSlideBox.getDOMNode().id).toEqual('test_id');
+    expect(getByRole('slider').id).toEqual('test_id');
 
     // properly associates label with custom id
-    const label = wrapper.find('label.test_label');
-    expect(label.prop('htmlFor')).toBe('test_id');
+    const label = container.getElementsByClassName('test_label')[0];
+    expect(label.getAttribute('for')).toBe('test_id');
   });
 
   it('can set id via buttonProps', () => {
     // Not the recommended way of doing things, but it should work consistently still
-    wrapper = mount(<Slider buttonProps={{ id: 'test_id' }} styles={{ titleLabel: 'test_label' }} />);
+    const { container, getByRole } = render(
+      <Slider buttonProps={{ id: 'test_id' }} styles={{ titleLabel: 'test_label' }} />,
+    );
 
-    const sliderSlideBox = wrapper.find('.ms-Slider-slideBox');
-    expect(sliderSlideBox.getDOMNode().id).toEqual('test_id');
+    expect(getByRole('slider').id).toEqual('test_id');
 
     // properly associates label with custom id
-    const label = wrapper.find('label.test_label');
-    expect(label.prop('htmlFor')).toBe('test_id');
+    const label = container.getElementsByClassName('test_label')[0];
+    expect(label.getAttribute('for')).toBe('test_id');
   });
 
   it('handles zero default value', () => {
     const slider = React.createRef<ISlider>();
 
-    wrapper = mount(<Slider defaultValue={0} min={-100} max={100} componentRef={slider} />);
+    render(<Slider defaultValue={0} min={-100} max={100} componentRef={slider} />);
     expect(slider.current!.value).toEqual(0);
   });
 
   it('handles zero value', () => {
     const slider = React.createRef<ISlider>();
 
-    wrapper = mount(<Slider value={0} min={-100} max={100} componentRef={slider} />);
+    render(<Slider value={0} min={-100} max={100} componentRef={slider} />);
     expect(slider.current!.value).toEqual(0);
   });
 
   it('calls onChange and onChanged when slider value changes with mouse', () => {
     const onChange = jest.fn();
     const onChanged = jest.fn();
     const slider = React.createRef<ISlider>();
-    wrapper = mount(<Slider onChange={onChange} defaultValue={5} onChanged={onChanged} componentRef={slider} />);
+    const { container, getByRole } = render(
```

Comment:
I wonder if this would be a good candidate for doing an additional test in cypress to ensure it works in real scenarios as well. But that's probably lower priority if the jest test is reliable.